source/administration/batch-framework-job-replicate.rst

```diff
@@ -0,0 +1,261 @@
+- The job provides for retry attempts in event that objects do not replicate
+- Batch jobs are one-time, curated processes allowing for fine control replication
+- (MinIO to MinIO only) The replication process copies object versions from source to target
+
+.. versionchanged:: MinIO Server RELEASE.2023-02-17T17-52-43Z
+
+   Run batch replication with multiple workers in parallel by specifying the :envvar:`MINIO_BATCH_REPLICATION_WORKERS` environment variable.
+
+Starting with the MinIO Server ``RELEASE.2023-05-04T21-44-30Z``, the other deployment can be either another MinIO deployment or any S3-compatible location using a realtime storage class.
+Use filtering options in the replication ``YAML`` file to exclude objects stored in locations that require rehydration or other restoration methods before serving the requested object.
+Batch replication to these types of remotes uses ``mc mirror`` behavior.
+
+Behavior
+--------
+
+Access Control and Requirements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Batch replication shares similar access and permission requirements as :ref:`bucket replication <minio-bucket-replication-requirements>`.
+
+The credentials for the "source" deployment must have a policy similar to the following:
+
+.. literalinclude:: /extra/examples/ReplicationAdminPolicy.json
+   :class: copyable
+   :language: json
+
+The credentials for the "remote" deployment must have a policy similar to the following:
+
+.. literalinclude:: /extra/examples/ReplicationRemoteUserPolicy.json
+   :class: copyable
+   :language: json
+
+
+See :mc:`mc admin user`, :mc:`mc admin user svcacct`, and :mc:`mc admin policy` for more complete documentation on adding users, access keys, and policies to a MinIO deployment.
+
+MinIO deployments configured for :ref:`Active Directory/LDAP <minio-external-identity-management-ad-ldap>` or :ref:`OpenID Connect <minio-external-identity-management-openid>` user management can instead create dedicated :ref:`access keys <minio-idp-service-account>` for supporting batch replication.
+
+Filter Replication Targets
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The batch job definition file can limit the replication by bucket, prefix, and/or filters to only replicate certain objects.
+The access to objects and buckets for the replication process may be restricted by the credentials you provide in the YAML for either the source or target destinations. 
+
+.. versionchanged:: MinIO Server RELEASE.2023-04-07T05-28-58Z
+
+   You can replicate from a remote MinIO deployment to the local deployment that runs the batch job.
+
+For example, you can use a batch job to perform a one-time replication sync to push objects from a bucket on a local deployment at ``minio-local/invoices/`` to a bucket on a remote deployment at ``minio-remote/invoices``.
+You can also pull objects from the remote deployment at ``minio-remote/invoices`` to the local deployment at ``minio-local/invoices``.
+
+Batch Compression
```

Comment:
@vadmeste

Reply:
Maybe we should call this, 'small files replication optimization' ?